### DISCUSSION TITLE
Want to help out and speak German? Searching German translator for website

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#1590] [Announcements]
You can help keeping the [Website](https://jamulus.io/de/) up to date in German!

## What is it?

A lot of German speaking people are using Jamulus. Also the documentation is (at least) partly translated to German. Until now, I've translated most of the content on my own but will not be able to translate the documentation for the upcoming release.

Therefore, we are searching for a volunteer (team) which takes over the German translation of the site. 

## What should you know?

1. Translating the website differs largely from translating the app. Since the site contains a lot of text, surely, it is a big, but manageable task (I translated v3.7.0 in about one evening with the help of Deepl). 
2. Until now I only translated the most important webpages (homepage, getting started, installation guides on all 3/4 OS) since all others probably don't get that much traffic.
3. Translation should sound natural and not differ too much from the English base. The German translation isn't that strict at the moment. However, this can be changed of course.
4. Have a look at the README file in the documentation on how to get started: https://github.com/jamulussoftware/jamuluswebsite#translating

## How do I start?

First of all, comment here. Afterwards, you can already have a look at the website and fix typos. Then wait for a ping, and start translating!

## I need help. Can anybody help me?

Yes, sure. There are and have been many translators for e.g. French (@trebmuh @jujudusud) and Spanish (@ignotus666). They should know a lot about the translation process. Moreover, @gilgongo (the "documentation guy") knows a lot about the English page (and all the management behind the website). If I have time, I might also guide you ;-).

**Comment by ewarning** [comment]
Hello,
I would like to help with the german translation.
Liebe Grüße

**Reply by hoffie** [reply]
That's great to hear! :)
We're just in the process of starting website translations for 3.8.0. We'll likely create tracking issues today and I'll ping you there. :)

**Comment by Helondeth** [comment]
Hey :)
I am also interested in helping with the translation.

**Reply by hoffie** [reply]
Great! Will ping you as well. There will likely be multiple files to be translated, so this work should be splittable. :)

**Comment by hoffie** [comment]
https://github.com/jamulussoftware/jamuluswebsite/issues/456 has been opened now.
